CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.13)
 
 project(PiP)
 
-add_compile_options(-fobjc-arc -Wno-deprecated-declarations -Wno-format)
+add_compile_options(-fobjc-arc -Wno-deprecated-declarations -Wno-format -mmacosx-version-min=14.0)
```

Comment:
Does it not build successfully for you without explictily adding that flag?
If it does not then add `set(CMAKE_OSX_DEPLOYMENT_TARGET "12.0.0")` as I still use monterey and remove the `macosx-version-min` flag.